dist/obs-server.spec

```diff
@@ -342,6 +342,8 @@ export OBS_VERSION="%{version}"
 DESTDIR=%{buildroot} make install FILLUPDIR=%{_fillupdir}
 if [ -f %{_sourcedir}/open-build-service.obsinfo ]; then
     sed -n -e 's/commit: \(.\+\)/\1/p' %{_sourcedir}/open-build-service.obsinfo > %{buildroot}/srv/www/obs/api/last_deploy
+else
+    echo "%version" > %{buildroot}/srv/www/obs/api/last_deploy
```

Comment:
This is wrong because it gets written into the commit tag but it is not the commit! The try should have been enough!